coalib/parsing/Filter.py

```diff
@@ -0,0 +1,22 @@
+from coalib.parsing.filters.LanguageFilter import language_filter
+
+
+class Filter:
+    all_filters = {'language': language_filter}
```

Comment:
I think I'll first complete rest, and then work on this one. This sounds like an interesting feature. But may require a bit more design.